docs/general/dql/selects.rst

```diff
@@ -834,6 +834,42 @@ Object property can also be addressed in the :ref:`where clause
     +------------+----------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+For selecting records having values with empty or unassigned objects, like::
+
+    cr> insert into locations (id, name, position, kind, inhabitants) values
+    ... (17, 'Voidspace 2', 0, 'Milliways', {});
+    INSERT OK, 1 row affected (... sec)
+
+.. Hidden: refresh locations
+
+    cr> refresh table locations;
+    REFRESH OK, 1 row affected (... sec)
+
+a suitable expression to match for both ``NULL`` values and empty objects, is::
+
+    cr> select count(*) from locations where
+    ... inhabitants is null or
+    ... inhabitants = {};
```

Comment:
`object_col = {}` does not match empty objects. But `IS NULL` does. So I'd think we don't need to document this for objects as  `IS NULL` behaves like expected.

Reply:
I've changed the statement to only use the `IS NULL` match.